public/friendlycode/js/fc/ui/bramble-project.js

```diff
@@ -0,0 +1,13 @@
+define(["jquery.min"], function($) {
+  "use strict";
+
+  function updateMeta(project) {
+    // Update the project title in the userbar
+    // see views/userbar.html
+    $("#project-title").text(project.title || "New Project");
```

Comment:
Same deal here, better to have `Constants.js` and reuse it everywhere.  Down the road we'll also need to ponder how we localize this.